package.json

```diff
@@ -122,7 +117,10 @@
       "no-use-before-define": "off",
       "quotes": 0,
       "space-before-function-paren": 0
-    }
+    },
+    "ignorePatterns": [
+      "**/*.snap"
+    ]
```

Comment:
Updated prettier, and it started included snap files. We don't care since they are auto generated